Kickstarter-iOS/Views/Controllers/ManagePledgePaymentMethodView.swift

```diff
@@ -0,0 +1,147 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class ManagePledgePaymentMethodView: UIView {
+  // MARK: - Properties
+
+  private lazy var adaptableStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var cardImageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var cardLabelsStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var expirationDateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var lastFourLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+
+  private let viewModel = CreditCardCellViewModel()
```

Comment:
We've been inconsistent in this lately but shouldn't we define `vm` as CreditCardCellViewModelType` before we initialize it? This way we will be told by the compiler if we try to access any variables directly (skipping `inputs`/`outputs`).

Could you maybe also add refactoring ticket in JIRA to do the same for all the other 17 occurrences of `let viewModel = ` where we let the compiler infer the type without being explicitly told so?